Shared/SentryIntegration.swift

```diff
@@ -79,7 +85,7 @@ public class SentryIntegration {
 
         let event = Event(level: severity)
         event.message = message
-        event.tags = ["event": tag]
```

Comment:
Is this going to give us trouble when mixing existing and new reports? In terms of search queries? I understand it is incorrect but maybe leave it?